README.md

```diff
@@ -21,7 +21,7 @@ rust:
 ```
 You should then be able to then use:
 ```
-multirust override nightly-2016-03-07
+rustup override add nightly-2016-03-07
```

Comment:
Well, let's wait until the "WARNING: This is an early beta. Expect breakage." disappears.